packages/atomic/src/components/search/atomic-breadbox/atomic-breadbox.tsx

```diff
@@ -36,6 +36,7 @@ const ELLIPSIS = '...';
 /**
  * The `atomic-breadbox` component creates breadcrumbs that display a summary of the currently active facet values.
  *
+ * @part container - The container of the component.
```

Comment:
I think we should mention here that this container surrounds the label and breadcrumb list.